NSDirectionalRectEdge+Utilities.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  NSDirectionalRectEdge+Utilities.swift
+//  ADUtils
+//
+//  Created by Gaétan Zanella on 23/11/2021.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+@available(tvOSApplicationExtension 13.0, *)
```

Comment:
C'est pas plutôt ca la syntaxe ?
```suggestion
@available(iOS 13.0, tvOS 13.0, *)
```